cfgov/jinja2/v1/job-description-page/index.html

```diff
@@ -47,87 +50,134 @@ <h1>{{ page.title }}</h1>
                 {% endif %}
                 ${{ '{:,.0f}'.format( page.salary_min ) }}–${{ '{:,.0f}'.format( page.salary_max ) }}
                 <a class="a-link 
-                          a-link__jump 
-                          a-link__icon 
+                          a-link__icon
                           cf-icon 
                           cf-icon__after 
                           cf-icon-download 
                           pay-grade-link"
                    href="http://files.consumerfinance.gov/f/documents/cfpb_pay_scales.pdf">
-                    <span class="a-link_text">See information on grades and base pay ranges</span>
+                   <span class="a-link_text">See information on grades and base pay ranges</span>
```

Comment:
This indentation was correct before. The `href` attr is only three spaces because it's aligned to the `class` attr